src/wolfgame.py

```diff
@@ -3517,7 +3516,7 @@ def transition_day(cli, gameid=0):
             victims.append(v)
         elif v.nick in var.ROLES["bodyguard"] and v.nick in angel.GUARDED and users._get(angel.GUARDED[v.nick]) in victims_set: # FIXME
             vappend.append(v)
-        elif v.nick in var.ROLES["harlot"] and harlot.VISITED.get(v.nick) is not None and users._get(harlot.VISITED[v.nick]) in victims_set: # FIXME
+        elif v in harlot.VISITED and harlot.VISITED[v] in victims_set:
```

Comment:
why not "harlot.VISITED.get(v) in victims_set"?

Reply:
Probably because I converted 1:1 and that's how it used to be.